### PR TITLE
Issues reported about overriding styles

### PR DESCRIPTION
**Pull request by elopezanaya**
### Description
_Include a description of the problem to be solved_

Suddenly a visual test started to fail, after validation it was confirmed as legitimate error and double checked with reports from users

https://github.com/microsoft/omnichannel-chat-widget/issues/543
https://github.com/microsoft/omnichannel-chat-widget/issues/462

## Solution Proposed
_Detail what is the solution proposed, include links to design document if required or any other document required to support the solution_

### Acceptance criteria
_Define what are the conditions to consider the PR has achieved the intended goal_

## Test cases and evidence
_Include what tests cases were considered, any evidence of testing for future references, to identify any corner cases, etc_

image

### Sanity Tests
-   [ ] You have tested all changes in Popout mode
-   [ ] You have tested all changes in cross browsers i.e Edge, Chrome, Firefox, Safari and mobile devices(iOS and Android)
-   [x] Your changes are included in the [CHANGELOG](../CHANGE_LOG.md)

## A11y
- [ ] You have run the [Automated Accessibility Check](https://accessibilityinsights.io/docs/en/windows/getstarted/automatedchecks) and have no reported issues

__*Please provide justification if any of the validations has been skipped.*__

### CODE REVIEW

**Review by Sonam-Ligade — APPROVED** [review]

File: chat-widget/src/components/livechatwidget/common/chatDisconnectHelper.ts [hunk 1]
```diff
@@ -26,6 +26,8 @@ const handleChatDisconnect = (props: ILiveChatWidgetProps, state: ILiveChatWidge
             });
```
Comment (mithunjohny) [inline chat-widget/src/components/livechatwidget/common/chatDisconnectHelper.ts:26]: Do we need to do the same logic in "true" switch case

Comment (elopezanaya) [reply chat-widget/src/components/livechatwidget/common/chatDisconnectHelper.ts:26]: apparently not
